simtools/camera_efficiency.py

```diff
@@ -0,0 +1,379 @@
+        if self._telescopeModel.hasParameter('camera_transmission'):
+            cameraTransmission = self._telescopeModel.getParameter('camera_transmission')
+
+        # cmd -> Command to be run at the shell
+        cmd = str(self._simtelSourcePath.joinpath('sim_telarray/bin/testeff'))
+        cmd += ' -nm -nsb-extra'
+        cmd += ' -alt {}'.format(self._telescopeModel.getParameter('altitude'))
+        cmd += ' -fatm {}'.format(self._telescopeModel.getParameter('atmospheric_transmission'))
+        cmd += ' -flen {}'.format(focalLength * 0.01)  # focal lenght in meters
+        cmd += ' -fcur {}'.format(RADIUS_CURV[self._telescopeModel.telescopeType])
+        cmd += ' {} {}'.format(pixelShapeCmd, pixelDiameter)
+        cmd += ' -fmir {}'.format(self._telescopeModel.getParameter('mirror_list'))
+        cmd += ' -fref {}'.format(self._telescopeModel.getParameter('mirror_reflectivity'))
+        if mirrorClass == 2:
+            cmd += ' -m2'
+        cmd += ' -teltrans {}'.format(self._telescopeModel.getTelescopeTransmissionParameters()[0])
+        cmd += ' -camtrans {}'.format(cameraTransmission)
+        cmd += ' -fflt {}'.format(self._telescopeModel.getParameter('camera_filter'))
+        cmd += ' -fang {}'.format(self._telescopeModel.camera.getFunnelEfficiencyFile())
+        cmd += ' -fwl {}'.format(self._telescopeModel.camera.getFunnelWavelengthFile())
+        cmd += ' -fqe {}'.format(self._telescopeModel.getParameter('quantum_efficiency'))
+        cmd += ' {} {}'.format(200, 1000)  # lmin and lmax
+        cmd += ' {} 1 {}'.format(300, self._zenithAngle)  # Xmax, ioatm, zenith angle
+        cmd += ' 2>{}'.format(self._fileLog)
+        cmd += ' >{}'.format(self._fileSimtel)
+
+        # Moving to sim_telarray directory before running
+        cmd = 'cd {} && {}'.format(self._simtelSourcePath.joinpath('sim_telarray'), cmd)
+
+        self._logger.info('Running sim_telarray with cmd: {}'.format(cmd))
+        os.system(cmd)
+        return
+    # END of simulate
+
+    def analyze(self, export=True, force=False):
+        '''
+        Analyze camera efficiency output file and store the results in _results.
+
+        Parameters
+        ----------
+        export: bool
+            If True, results will be exported to a file automatically. Alternativelly, exportResults
+            function can be used.
+        force: bool
+            If True, existing results files will be removed and analysis will be done again.
+        '''
+        self._logger.info('Analyzing CameraEfficiency')
+
+        if self._fileResults.exists() and not force:
+            self._logger.info('Results file exists and force=False - skipping analyze')
+            self._readResults()
+            return
+
+        # List of parameters to be calculated and stored
+        effPars = [
+            'wl',
+            'eff',
+            'effAtm',
+            'qe',
+            'ref',
+            'masts',
+            'filt',
+            'funnel',
```

Comment:
I changed everywhere from funnel to pixel, to be a bit more accurate (in some cases). Can I change this one as well? I cannot see it being explicitly used anywhere, so I assume it's OK?

Reply:
Yes, please